doc/docs/Python_Tutorials/Basics.md

```diff
@@ -51,7 +51,9 @@ geometry = [mp.Block(mp.Vector3(mp.inf,1,mp.inf),
 
 The waveguide is specified by a `Block` (parallelepiped) of size $\infty \times 1 \times \infty$, with $ε=12$, centered at (0,0) which is the center of the cell. By default, any place where there are no objects there is air ($ε=1$), although this can be changed by setting the `default_material` variable. The resulting structure is shown below.
 
-<center>![](../images/Python-Tutorial-wvg-straight-eps-000000.00.png)</center>
+<p align="center">
+  <img  src="../images/Python-Tutorial-wvg-straight-eps-000000.00.png">
```

Comment:
Remove the extra space between `img` and `src`. This also applies to several other instances in this PR.